e2e/replay/msg_persister.go

```diff
@@ -12,17 +12,17 @@ import (
 
 const directoryPath = "../SavedState"
 
-// replayMessagePersister encapsulates logic for saving messages in .flow file
-type replayMessagePersister struct {
+// MessagePersister encapsulates logic for saving messages in .flow file
+type MessagePersister struct {
 	lock     sync.Mutex
-	messages []*ReplayMessage
+	messages []*Message
 	file     *os.File
 }
 
-// saveMetaData saves node meta data to .flow file
-func (r *replayMessagePersister) saveMetaData(nodeNames *[]string) error {
+// SaveMetaData saves node meta data to .flow file
+func (r *MessagePersister) SaveMetaData(nodeNames *[]string) error {
```

Comment:
Seems like this doesn't need to be public eventually (it is called in `MessageNotifier`, which is in the `replay` package as well)

Reply:
Done